apps/app/src/test/resources/include/scans/_scan.conf

```diff
@@ -12,4 +12,6 @@
   participant-client {
     ledger-api.auth-config.type = "none"
   }
+  # Reduce in tests as SVs change relatively frequently here.
+  sv-node-state-cache-ttl = 1s
```

Comment:
nit: would it make sense to create an extra layer called `cache` and then include `state-cache-ttl` in it?

Reply:
done